balancer/frontend-v2
# Remove default tokens list, change vetted to allowlisted

# Description

- Removed the old default tokens list as it's no longer used in the UI. In the future we may add a special list.
- Renamed the vetted list to allowlisted to be consistent with tokenlists repo and other places that we allow tokens. As vetted was ambiguous if it represents all allowlisted tokens or a hand picked "vetted" set of them.

This fixes the token list toggle in the token search modal, it now has the balancer allowlist permanently on and you can toggle other lists. Previously the vetted list was hidden and the listed-old toggle was displayed which didn't really do anything. 

### Future Considerations:

- Maybe we should move pool tokens to another list and allow them to be toggled on and off? As they look weird being in the normal swap list even though they are swappable tokens
- I'm working on generating a list of top 100 tokens by volume that we could have as the default list and then users can toggle on the full allowlist if they like. 


## Type of change


## How should this be tested?

- Load the token list modal on different networks, ensure it has all tokens and the token list is permanently enabled. 

## Checklist:
